src/Controller/Admin/DigitalResourceTypeDescriptorCrudController.php

```diff
@@ -72,10 +62,6 @@ public function configureCrud(Crud $crud): Crud
 
     /**
      * Configure fields for CRUD.
-     *
-     * @param string $pageName
```

Comment:
keep?

Reply:
Yes, if there was a good description that goes with it,